Src/Support/Google.Apis.Auth/TaskExtensions.cs

```diff
@@ -14,6 +14,7 @@
 limitations under the License.
 */
 
+using System;
```

Comment:
Revert this?

Reply:
Yep, doing on next push and merging on green.